.readthedocs.yaml

```diff
@@ -14,4 +14,8 @@ build:
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
   configuration: docs/source/conf.py
-  fail_on_warning: false
\ No newline at end of file
+  fail_on_warning: false
+
+python:
+  install:
+    - requirements: docs/requirements.txt
```

Comment:
I guess this will need change.

Reply:
Yes, the original aim was to remove `pip install` commands from inside notebooks. I hadn't realised these requirements could be moved into Poetry.